src/ctapipe/irf/preprocessing.py

```diff
@@ -189,44 +133,158 @@ def make_empty_table(self) -> QTable:
                 " (e.g. gamma in gamma-ray analysis) is more likely",
             ),
             Column(
-                name="weight",
-                unit=u.dimensionless_unscaled,
-                description="Event weight",
+                name="weight", unit=u.dimensionless_unscaled, description="Event weight"
             ),
-        ]
+        ],
+        help="Schema definition for output event QTable",
+    ).tag(config=True)
+
+    def __init__(self, config=None, parent=None, **kwargs):
+        super().__init__(config=config, parent=parent, **kwargs)
+        self.quality_query = EventQualityQuery(parent=self)
+
+    @property
+    def columns_to_rename(self) -> dict:
+        if self.disable_column_renaming:
+            return {}
+
+        default = {
+            f"{self.energy_reconstructor}_energy": "reco_energy",
+            f"{self.geometry_reconstructor}_az": "reco_az",
+            f"{self.geometry_reconstructor}_alt": "reco_alt",
+            f"{self.gammaness_classifier}_prediction": "gh_score",
+            "subarray_pointing_lat": "pointing_alt",
+            "subarray_pointing_lon": "pointing_az",
+        }
+        return {**default, **self.columns_to_rename_override}
```

Comment:
Regarding my above thread, this only extends the default renaming and does not overwrite it, no?
Do you want to do something like this:
```python
if self.columns_to_rename_override is None:
    return default
else:
    return self.columns_to_rename_override
```